config/import.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Import
+    |--------------------------------------------------------------------------
+    |
+    | This file is a custom addition for setting constants used to import data.
+    |
+    */
+    'group_types' => [
+        'college_board' => [
+            'filter_by_location' => true,
+            'name' => 'The College Board',
+            'path' => 'https://gist.githubusercontent.com/aaronschachter/2933b5d29bbee4a866de2de49a1f35f7/raw/452d267228d2478de6e845982291c7510cab08cf/groups.csv',
```

Comment:
One really strange thing happening when I copy/paste the raw CSV into a Gist is that it seems to be [adding extra rows](https://dosomething.slack.com/archives/CTVPG6L4R/p1597244201343900?thread_ts=1597086544.318300&cid=CTVPG6L4R). It might not be the most reliable place to store a file with 15k rows.

Is there an easy process for getting a raw file over to amazon AWS? There are pros and cons of hardcoding these URLs here, ideally the file path shouldn't change -- this have always been one-time imports, but I run them regularly for testing/dev.

Reply:
Woah, spooky! I've had good luck providing files over standard input (for example in [this command](https://github.com/DoSomething/northstar/blob/master/app/Console/Commands/DeleteUsersCommand.php)). Then you can just "pipe" the file into the command from your local machine & don't have to worry about hosting it somewhere on the internet:

```sh
cat groups.csv | heroku run php artisan rogue:groups-import
```

Reply:
For larger files, we could also definitely drop it in this app's S3 bucket but that's a bit more of a hassle.

Reply:
Thanks for the help! This did the trick.